From b7b285ee9e05102f770d41643e5534bade289a58 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Timo=20Ter=C3=A4s?= <timo.teras@iki.fi>
Date: Mon, 21 Mar 2016 11:31:18 +0200
Subject: [PATCH] nlplug-findfs: increase the /sys recursion limit

certain platform and usb devices expose things deep down the tree,
increase the recursion limit
---
 nlplug-findfs.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/nlplug-findfs.c b/nlplug-findfs.c
index 5bdb7b1..b11b7b8 100644
--- a/nlplug-findfs.c
+++ b/nlplug-findfs.c
@@ -805,9 +805,9 @@ static void *trigger_thread(void *data)
 	};
 	char path[PATH_MAX] = "/sys/bus";
 
-	recurse_dir(path, &opts, 8);
+	recurse_dir(path, &opts, 64);
 	strcpy(path, "/sys/devices");
-	recurse_dir(path, &opts, 8);
+	recurse_dir(path, &opts, 64);
 	write(fd, &ok, sizeof(ok));
 	return NULL;
 }
-- 
2.7.4

